### PR TITLE
Implement 'customSignerURL' feature

### PR DESCRIPTION
**Pull request by vitaliipustovyi**

### CODE REVIEW

**Review by dmitriiIdentityLabs — APPROVED** [review]

**Review by dmitriiIdentityLabs — APPROVED** [review]

File: packages/identitykit/src/libs/react/modal.tsx [hunk 1]
```diff
@@ -27,48 +53,117 @@ export const IdentityKitModal = (props: { theme: IdentityKitTheme }) => {
...
+                    />
+                    <p className="text-sm text-black dark:text-white">{signer.label}</p>
+                  </div>
+                ))}
+                {!isViewAll ? (
+                  <Button type="secondary" onClick={() => setIsViewAll(true)}>
+                    View all
+                  </Button>
+                ) : (
+                  <div>
```
Comment (chorniy8pavlo) [inline packages/identitykit/src/libs/react/modal.tsx:134]: I would put this in a separate react component

File: packages/identitykit/src/libs/react/modal.tsx [hunk 2]
```diff
@@ -27,48 +53,117 @@ export const IdentityKitModal = (props: { theme: IdentityKitTheme }) => {
...
+                    <div className="text-l text-black font-bold my-[20px]">Custom Connect</div>
```
Comment (chorniy8pavlo) [inline packages/identitykit/src/libs/react/modal.tsx:135]: text-l ?

File: package.json [hunk 3]
```diff
@@ -40,5 +42,8 @@
     "docs",
     "examples/*",
     "packages/*"
-  ]
-}
\ No newline at end of file
+  ],
+  "dependencies": {
+    "react-hook-form": "^7.52.1"
```
Comment (chorniy8pavlo) [inline package.json:47]: I think all dependencies you've added should be in packages/identitykit/package.json.
Not in the root.

**Review by chorniy8pavlo — CHANGES_REQUESTED** [review]

File: packages/identitykit/src/libs/react/ui/input/index.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import { useContext } from "react"
+import { isValidURL } from "../../utils"
+import { useForm } from "react-hook-form"
+import { Button } from "../button"
+import { IdentityKitContext } from "../../context"
+
+export const CustomSignerInput = () => {
```
Comment (chorniy8pavlo) [inline packages/identitykit/src/libs/react/ui/input/index.tsx:7]: - Please put it on the same level as modal.tsx
- Rename file from `input.tsx` to `signer-input.tsx`

Reason for these - it's not a reusable input component it's a specific feature.

**Review by chorniy8pavlo — DISMISSED** [review]

**Review by chorniy8pavlo — APPROVED** [review]

**Pull request merged**